### PR TITLE
DTGB-553: Add crisis communication block to styleguide

### PR DESCRIPTION
**Pull request by hstandaert**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.crisis-communication-block {
+  p {
+    margin: 0;
+    font-weight: 700;
```
Comment (delrueba) [inline styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss]: I believe this is too bold, it's 600 in design.

File: styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.crisis-communication-block {
+  p {
+    margin: 0;
```
Comment (delrueba) [inline styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss]: It's better not to override the atoms too much.
I understand design only has a margin of 16px and we have 20px... but that's simply the nature of a paragraph.
If a user decides to break up his warning in two paragraphs, you'd loose readability.

Don't be afraid to question these small inconsistencies.

File: styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    font-weight: 700;
+  }
+
+  .icon-list {
+    margin-top: .8rem;
```
Comment (delrueba) [inline styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss]: See paragraph

File: styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  }
+
+  ul:last-of-type {
+    margin-bottom: 0;
```
Comment (delrueba) [inline styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss]: Maybe we should move this to box level: *:last-of-type
@meirege ?

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss (hunk 2 above)
Comment (hstandaert) [reply styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss]: Understood! 👍I'll take it into account for future development.
Thanks for the feedback

File: styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss (hunk 4 above)
Comment (meirege) [reply styleguide/components/31-molecules/crisis-communication-block/_crisis-communication-block.scss]: This change breaks the whitespace between some components. Also this breaks the margin of the title...
A title has it's margin, I wouldn't change that here, we don't do it elsewhere so...

**Review by meirege — CHANGES_REQUESTED** [review]

**Review by meirege — APPROVED** [review]

**Review by delrueba — APPROVED** [review]

**Pull request merged**